src/banana/data/genpdf/Toy.info

```diff
@@ -0,0 +1,29 @@
+SetDesc: 'Toy PDF for benchmarking purpose as defined by Les Houches WG in arXiv:hep-ph/0204316v1'
```

Comment:
I'm almost sure this is a YAML file, so things could be broken even on multiple lines and so on.

However, if this is working never mind. No one is going to edit this file any longer.